input/blog/2020-12-10-cake-vscode-extension-release-1.0.0.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Cake VSCode Extension Release 1.0.0
+category: News
+author: gep13
+---
+
+The Cake Team are very happy to announce the 1.0.0 release of the [Cake Extension for VSCode](https://marketplace.visualstudio.com/items?itemName=cake-build.cake-vscode).  This release has been a long time in the making, but we are hoping that it will be worth the wait!
+
+With the upcoming release of Cake 1.0.0, and the transition in our documentation to recommend the usage of the .Net Tool Runner for Cake, the main focus of this version of the extension was to start using the .Net Tool Runner for all of the built in actions.  For example:
+
+* If you choose `Run Task` from the command palette, the chosen task will be executed using the .Net Tool Runner for Cake
+* If you use the code lens options to either `run task` or `debug task`, the .Net Tool Runner for Cake will be used
+
+:::{.alert .alert-info}
+**NOTE:**
```

Comment:
I would get rid of the **NOTE:** part here to be consistent with other annotation blocks